test-clients/src/main/java/com/hedera/services/bdd/suites/records/RecordCreationSuite.java

```diff
@@ -58,7 +58,10 @@ public static void main(String... args) {
 	private HapiApiSpec accountsGetPayerRecordsIfSoConfigured() {
 		return defaultHapiSpec("AccountsGetPayerRecordsIfSoConfigured")
 				.given(
-						cryptoCreate("payer")
+						cryptoCreate("payer"),
```

Comment:
LGTM. But, @anighanta had a PR open with changes to this Suite. Not sure if that is merged